data/benefits/aide-au-bafa-pour-une-session-d’approfondissement-ou-de-qualification-caf-du-val-de-marne.yml

```
@@ -0,0 +1,15 @@
+label: Aide au Bafa pour une session d’approfondissement ou de qualification
```

Même commentaire qu'ici #1948 (comment)
```suggestion
label: aide au Bafa pour une session d’approfondissement ou de qualification
```